### DISCUSSION TITLE
在寫衛星軌跡模擬的程式中 搞不懂np.array是什麼意思

### DISCUSSION THREAD

**Discussion by Chengbosco** [1101-GS4719-C/Python-Community-GS4719-C#1710] [Q&A]
最近在寫衛星軌道的模擬程式，再有段實在打不出來(要用到二維陣列)，於是我問了gpt，但它打出來的東西我完全看不懂。
-------------------------------------------------------------------------------------------------------------------------------------
def rocket_trajectory(rocket_mass, planet_mass, iposition, ivelocity, tstep=0.2, totaltime=100999):
    time = np.arange(0, totaltime, tstep)
    positions = [np.array(iposition)]
    velocities = [np.array(ivelocity)]
    masses = [rocket_mass]
-------------------------------------------------------------------------------------------------------------------------------------
numpy.arange(start, stop, step)這段還看得懂，但positions = [np.array(iposition)]中array(ipotition)是指什麼，[https://www.youtube.com/watch?v=4xm8dyYIBSk]這篇也只說這樣可以製造陣列，但到底製造了什麼也搞不清楚。

麻煩大家了

**Comment by ATREE01** [comment]
對於套件內的函式有所不懂的時候可以去察看官方的document [numpy.array](https://numpy.org/doc/stable/reference/generated/numpy.array.html)
上面也有提供一些範例，簡單來說就是把其他的陣列轉換成np的陣列讓你可以使用一些包含在numpy中的功能。
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/100257906/1e49d95a-4f86-4fda-9315-41dd8c360f67)
